src/DDoSX/CdnClient.php

```diff
@@ -13,7 +14,13 @@ class CdnClient extends BaseClient
      */
     protected $basePath = 'ddosx/';
 
-    protected $requestMap = [];
+    const CDN_MAP = [];
+
+    const RULES_MAP = [
+        'cache_control' => 'cacheControl',
+        'mime_types' => 'mimeTypes',
+        'cache_control_duration' => 'cacheControlDuration'
+    ];
 
     /**
      * @param Cdn $cdn
```

Comment:
the $response encoding on line 33 still has `$this->requestMap` instead of `self::CDN_MAP`